courses/free-courses-it.md

```diff
@@ -62,6 +62,7 @@
 ### C
 
 * [Corso di programmazione in C](https://www.youtube.com/playlist?list=PLO4y9a8lTpK2ugwPRLN_1oOrlzX9Zc9It) - R. Rizzi (Università di Verona)
+* [Corso di programmazione in C (Standard C89/C90)](https://staff.polito.it/claudio.fornaro/CorsoC/) - C. Fornaro (Politecnico di Torino)
```

Comment:
It's a course :+1: [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#books-vs-courses)